worker/worker/request_handlers.go

```diff
@@ -361,6 +366,11 @@ func HandleUploadRestReq(uploadKey string, localFile string) {
 		}
 	}
 
+	err = transcodeAudio(&c)
+	if err != nil {
+		log.WithError(err).Error("Error transcoding audio")
+	}
```

Comment:
Depending on your use case with the audio, is uploading to the edge server for streaming required?

Reply:
Eventually audio files should be served the same way as video files are. So, I guess yes? Maybe we can discuss this in the next meeting.

Of course we still need to implemented a proper audio player page on the front end (Ultimately I don't think reusing the watch page as audio page is a good idea 😅)